src/main/java/com/nhantran/models/User.java

```
@@ -50,4 +55,50 @@ public String getPid() {
     public void setPid(String pid) {
         this.pid = pid;
     }
+
+    private static User getLoginAccountFromJsonFile(String accountName) {
+        JSONObject jsonObject = JsonFileReader.readJsonFromFile(Constants.ACCOUNT_JSON_FILE_PATH);
+        if (jsonObject != null) {
+            JSONObject accounts = (JSONObject) jsonObject.get("loginAccounts");
+            JSONObject userObj = (JSONObject) ((JSONArray) accounts.get(accountName)).get(0);
+            return new User((String) userObj.get("email"), (String) userObj.get("password"));
+        } else {
+            return null;
```

if null throw exception instead of null